frontend/src/lib/candidate/components/PasswordSetComponent/PasswordSetComponent.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+  import {t} from '$lib/i18n';
+  import {Button} from '$lib/components/button';
+  import PasswordField from '../PasswordField/PasswordField.svelte';
+  import {getRoute, Route} from '$lib/utils/navigation';
+  import PasswordValidator from '../passwordValidator/PasswordValidator.svelte';
+  import {createEventDispatcher} from 'svelte';
+  export let password1 = '';
+  export let password2 = '';
+  export let autocomplete1 = '';
+  export let autocomplete2 = '';
+  export let errorMessage = '';
+  export let validPassword = false;
+
+  $: disableSetButton = validPassword && password2.length > 0;
+  //dispatcher used for a function that calls the submit button's function in parent component
+  const dispatch = createEventDispatcher();
+  export function ButtonPressed() {
+    dispatch('ButtonPressed');
+  }
+</script>
+
+<!--
+@component
+PasswordSetComponent is a component used by PasswordSetPage and PasswordResetPage, that contains a password input field,
+a second confirmation password input field, a button for submitting the password, a button to contact support and informatory text
+
+### Properties
+
+- `password1` : variable the first password input box uses for its value
+- `password2` : variable the second password input box uses for its value
```

Comment:
These should be named `password` and `passwordConfirmation`. The other props should be named similarly. You also don't need to specify that the props are variables: the description should be as concise as possible. E.g. "`password`: value for the password field"